### ISSUE TITLE
Set `env_bytes_limit_in_kb` and `env_list_bytes_limit_in_kb` to `0` make other scripts fail

### ISSUE THREAD

**Issue by stephanecopin** [bitrise-io/envman#148]
In our workflow, before `envman` got updated to `2.0.0`, we were doing in a `Script` step:
```
mkdir -p ~/.envman
echo -e "{\n\t\"env_bytes_limit_in_kb\": 0,\n\t\"env_list_bytes_limit_in_kb\": 0\n}" > ~/.envman/configs.json
```
And the rest of the workflow executed as expected, and we could set environment variable of arbitrary size via `envman` (which is a requirement of our workflow, as some of our generated environment variables can get quite big).
After the `envman` update to `2.0.0`, the same `Script` still succeed, but then any other `Script` step that executes afterwards fail with the following error message:
```
+------------------------------------------------------------------------------+
| (X) <Script Step>                                                            |
+------------------------------------------------------------------------------+
| id: script                                                                   |
| version: 1.1.5                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: bash                                                                |
| time: 2018-09-17T06:11:24-07:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
 [!] => Failed: No script (content) defined for execution!
|                                                                              |
```
@viktorbenei has a reproducing example from one of our build, that I won't link here.

**Comment by viktorbenei** [comment]
Note: as @stephanecopin mentioned in our Slack chat it seems the code should allow the `0` limit and was not changed in the v2 release of `envman` : https://github.com/bitrise-io/envman/blob/master/cli/add.go#L42-L62

**Comment by godrei** [comment]
Hi @stephanecopin thanks for the report, we scheduled to fix this.

**Comment by godrei** [comment]
Hi @stephanecopin ,
we released a new version [2.1.0](https://github.com/bitrise-io/envman/releases/tag/2.1.0) which restores the env limits previous (before 2.0.0) functionality.
